docs/run-keep-ecdsa.adoc

```diff
@@ -323,6 +323,64 @@ reference].
 |No
 |===
 
+==== Example BeneficiaryAddresses
+
+[%header,cols=2*]
+|===
+|Description
+|Address
+|mainnet xpub extended public key
+|`BeneficiaryAddress = "xpub6Cg41S21V____REPLACE_WITH_VALID_DATA____vZvH8XyNzunEqLdk9dxyXQUoy7ALWQFNn5K1me74aEMtS6pUgNDuCYTTMsJzCAk9sk1"`
+
+|mainnet ypub extended public key
+|`BeneficiaryAddress = "ypub6Xxan668a____REPLACE_WITH_VALID_DATA____gaxnBh44HpkTi2TTkm1u136qjUxk7F3jGzoyfrGpHvALMgJgbF4WNXpoPu3QYrqogMK"`
+
+|mainnet zpub extended public key
+|`BeneficiaryAddress = "zpub6rePDVHfR____REPLACE_WITH_VALID_DATA____ykG47mZYyGTjsuq6uzQYRakSrHyix1YTXKohag4GDZLcHcLvhSAs2MQNF8VDaZuQT9"`
+
+|testnet3/regtest tpub extended public key
+|`BeneficiaryAddress = "tpubDEXzoXkNd____REPLACE_WITH_VALID_DATA____2erw2yS3VgY5EoYdcKH24VPqeAgBTF6i82Ft9NG1iVjSQVAvFBfd2wkRQXF1W2Q8W1"`
+
+|testnet3/regtest upub extended public key
+|`BeneficiaryAddress = "upub5DnYQWgCD____REPLACE_WITH_VALID_DATA____JnHCp5BwyvM7Xm7S5r5n5ZYMJ1WrrM31i4kcsWwW2vxcJS1kfsuKgK9vME2z1cx6aX"`
+
+|testnet3/regest vpub extended public key
+|`BeneficiaryAddress = "vpub5Zx5difzi____REPLACE_WITH_VALID_DATA____WHFTpF7pJqnyAnADhbVrFYX7eCK8V2WBBVprxzJrSk15NsYHiB8CvV8h4JnXkU66as"`
+
+|mainnet P2PKH btc address
+|`BeneficiaryAddress = "1MjCqoLqMZ6Ru____REPLACE_WITH_VALID_DATA____64TTtP16XnpSdiE8Kpgcx"`
+
+|testnet P2PKH btc address
+|`BeneficiaryAddress = "mkHS9ne12qx9p____REPLACE_WITH_VALID_DATA____S9VojpwU5xtRd4T7X7ZUt"`
+
+|mainnet P2SH btc address
+|`BeneficiaryAddress = "3J98t1WpEZ73C____REPLACE_WITH_VALID_DATA____NmQviecrnyiWrnqRhWNLy"`
+
+|mainnet Bech32 (segwit) P2WPKH btc address
+|`BeneficiaryAddress = "bc1qar0srrr7x____REPLACE_WITH_VALID_DATA____fkvy5l643lydnw9re59gtzzwf5mdq"`
+
+|mainnet Bech32 (segwit) P2WPSH btc address
+|`BeneficiaryAddress = "bc1qrp33g0q5c____REPLACE_WITH_VALID_DATA____cefvpysxf3qccfmv3"`
+
+|mainnet P2PK compressed btc public key (0x02)
+|`BeneficiaryAddress = "02192d74d0cb9____REPLACE_WITH_VALID_DATA____c3a957724895dca52c6b4"`
+
+|mainnet P2PK compressed btc public key (0x03)
+|`BeneficiaryAddress = "03b0bd634234a____REPLACE_WITH_VALID_DATA____37f23c2c409273eb16e65"`
+
+|mainnet P2PK compressed btc public key (0x04)
+|`BeneficiaryAddress = "0411db93e1dcd____REPLACE_WITH_VALID_DATA____1482ecad7b148a6909a5cb2e0eaddfb84ccf9744464f82e160bfa9b8b64f9d4c03f999b8643f656b412a3"`
+
+|testnet P2SH btc address
+|`BeneficiaryAddress = "2NBFNJTktNa7G____REPLACE_WITH_VALID_DATA____ZusGbDbGKRZTxdK9VVez3n"`
+
+|testnet Bech32 (segwit) P2WPKH btc address
+|`BeneficiaryAddress = "tb1qw508d6qej____REPLACE_WITH_VALID_DATA____xtdg4y5r3zarvary0c5xw7kxpjzsx"`
+
+|testnet P2PK compressed btc public key (0x02)
+|`BeneficiaryAddress = "02192d74d0cb9____REPLACE_WITH_VALID_DATA____4344c9569c2e77901573d8d7903c3ebec3a957724895dca52c6b4"`
+|===
```

Comment:
We could remove `mainnet` prefix from all public key samples as the network is irrelevant for the value. With this change we should remove `testnet P2PK compressed btc public key (0x02)` case as this would duplicate `P2PK compressed btc public key (0x02)`.

```suggestion
| P2PK compressed btc public key (0x02)
|`BeneficiaryAddress = "02192d74d0cb9____REPLACE_WITH_VALID_DATA____c3a957724895dca52c6b4"`

| P2PK compressed btc public key (0x03)
|`BeneficiaryAddress = "03b0bd634234a____REPLACE_WITH_VALID_DATA____37f23c2c409273eb16e65"`

| P2PK compressed btc public key (0x04)
|`BeneficiaryAddress = "0411db93e1dcd____REPLACE_WITH_VALID_DATA____1482ecad7b148a6909a5cb2e0eaddfb84ccf9744464f82e160bfa9b8b64f9d4c03f999b8643f656b412a3"`

|testnet P2SH btc address
|`BeneficiaryAddress = "2NBFNJTktNa7G____REPLACE_WITH_VALID_DATA____ZusGbDbGKRZTxdK9VVez3n"`

|testnet Bech32 (segwit) P2WPKH btc address
|`BeneficiaryAddress = "tb1qw508d6qej____REPLACE_WITH_VALID_DATA____xtdg4y5r3zarvary0c5xw7kxpjzsx"`
```